File: Data Structures & Algorithms/redistribute-characters-to-make-all-strings-equal/submission-0.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
       int ans[26]={0};
       for(int i=0;i<words.size();i++){
        for(int j=0;j<words[i].size();j++){
            ans[words[i][j]-'a']++;
        }
       }
       int n=words.size();
       for(int i=0;i<26;i++){
        if(ans[i]%n!=0){
          return false;
        }
       }
       return true; 
    }
};

File: Data Structures & Algorithms/find-common-characters/submission-0.py
class Solution:
    def commonChars(self, words: List[str]) -> List[str]:
        
        s=Counter(words[0])
        for i in range(1,len(words)):
            s=s & Counter(words[i])
        result = []

        for ch in s:
            result.extend([ch] * s[ch])

        return result
        
       
        